src/routes/contents/[id]/+page.svelte

```diff
@@ -0,0 +1,229 @@
+<script>
+    import { page } from '$app/stores';
+    import Navbar from '$lib/components/Navbar.svelte';
+    import {
+        s,
+        t,
+        themeColors,
+        modal,
+        MODAL_COLLECTION,
+        convertStyle,
+        contentsStack
+    } from '$lib/data/stores';
+    import { base } from '$app/paths';
+    import { refs } from '$lib/data/stores';
+    import { goto } from '$app/navigation';
+    import config from '$lib/data/config';
+
+    $: highlightColor = $themeColors['ContentsItemTouchColor'];
+    $: title = setTitle($page);
+
+    function onClick(event, item) {
+        event.target.style.background = highlightColor;
+        setTimeout(() => {
+            clicked(item);
+        }, 100);
+    }
+
+    function checkImageSize(item) {
+        if (item.features['image-width']) {
+            return 'width: ' + item.features['image-width'];
+        } else {
+            //nothing here - default size
+        }
+    }
+    async function clicked(item) {
+        //check type of link
+        switch (item.linkType) {
+            //reference linkType
+            case 'reference':
+                contentsStack.pushItem($page.data.menu.id);
+                setReference(item);
+                goto(`${base}/`);
+                break;
+            //need to check if these are the correct strings to use
+            case 'screen':
+                //not handled yet
```

Comment:
Seems like it is handled now.  Remove comment? I guess I should have when I implemented it. ;-)